lib/nexthop.h

```diff
@@ -145,6 +146,10 @@ struct nexthop {
 
 	/* SR-TE color used for matching SR-TE policies */
 	uint32_t srte_color;
+
+	/* seg6local */
+	enum seg6local_action_t nh_seg6local_action;
+	struct seg6local_context *nh_seg6local_ctx;
```

Comment:
Do these need to be added to the `zapi_nexthop_from_nexthop()` paths in the zebra code? that is, if zebra has nexthops with this information, will that info need to be marshalled into corresponding zapi_nexthops to be sent to other daemons? or is this info that does not need to be sent from zebra to others?

Reply:
Yes thees do! 
In current PR, there is no case for that. but that situation should be supported!
I had not implemented `nexthop_from_zapi_nexthop` and `zapi_nexthop_from_nexthop`.

After this PR will be merged, my BGPd patch may uses this marchalling.

I'll add them.

Reply:
I fixed with this force-push. Also thank you for your great comment. 
https://github.com/FRRouting/frr/compare/92eca9a4f0f8532f301a8a824238adc3d11ec4d1..6edefaf0cffcfd9e6aaf6a67404ee27dfbf6a9a7